web/context_processors.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.views import LoginView
+from django.shortcuts import reverse
+
+
+def login(request):
+    # The current `next` parameter
+    current_redirect_url = LoginView(request=request).get_redirect_url()
+
+    # If at the login page:
+    if request.path.startswith(reverse('login').rstrip("/")):
```

Comment:
Is there a reason why some strings use single quotes?

Reply:
I'm used to referring to specific defined values (practically the same as enums) with single quotes and everything else with double-quotes.

To take this file as an example, `'login'` is the defined name of a URL, `"/"` is just a path (one could argue that it's borderline a defined value), `'login_next_param'` will become a defined value that's used in the template, and `"?next="` is a constructed string when combined with `login_redirect_path` (although I guess just `'next'` would be a defined value).

Ultimately, it's only for giving hints as to which values are defined elsewhere in the code, but otherwise not really important.